ecl_config/include/ecl/config/ecl.hpp

```diff
@@ -0,0 +1,163 @@
+/**
+ * @file /include/ecl/config/ecl.hpp
```

Comment:
is this a generated file?  seems like it is and shouldn't be included in the PR.

Reply:
Yes, looks like you've captured what to do correctly with it now there's no more devel folder (to CMAKE_BINARY_DIR).